### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by 807502278** [coolzilj/ComfyUI-LJNodes#5]
Your plugin is very useful, I'm here again,hahaha

Problem: The fifth one is very practical, but when I output "Right Click", the menu does not appear
Suggestion: Change the name of the group and node. It is recommended to use the same shortcut key. Now, I am using F2, but I will still double-click to rename the group

**Comment by coolzilj** [comment]
did you update to lastest version?

**Comment by 807502278** [comment]
Yes, I just git pull again, and there are still these two questions

I remember there was a plugin before, which was also right-click on the output. Some people could use it, while others couldn't

**Comment by liangkun43** [comment]
[https://github.com/ty0x2333/ComfyUI-Dev-Utils](url)
[https://github.com/tinyterra/ComfyUI_tinyterraNodes](url)

**Comment by coolzilj** [comment]
> [https://github.com/ty0x2333/ComfyUI-Dev-Utils](url) [https://github.com/tinyterra/ComfyUI_tinyterraNodes](url) When using these two plugins, the right-click menu does not appear

Thanks for the info. I’ll look into it.

**Comment by 807502278** [comment]
![image](https://github.com/coolzilj/ComfyUI-LJNodes/assets/46019416/3595dcdc-c010-493b-bb92-02ff204dd83d)
I removed ComfyUI-Dev-Utils and indeed the menu appeared, but there was no "Add Reroute in between" (although I did not install ComfyUI_tinyterraNodes, I don't know why the name "TTN" appeared in the menu)

**Comment by enternalsaga** [comment]
same problem but i found the add route node happens to show up on a few nodes, 
![image](https://github.com/coolzilj/ComfyUI-LJNodes/assets/31148570/c5a060e8-2a2c-43ec-8890-fd939f57df24)
![image](https://github.com/coolzilj/ComfyUI-LJNodes/assets/31148570/d834a08a-cecb-4840-b0dd-527b2f3eb59d)
most nodes wont do
![image](https://github.com/coolzilj/ComfyUI-LJNodes/assets/31148570/e3010b6f-7e2b-4016-8e5e-2b8efa391ea6)
![image](https://github.com/coolzilj/ComfyUI-LJNodes/assets/31148570/9b462078-6723-4696-9905-3e19181194a6)

**Comment by thefoxfarmer** [comment]
The context menu goes away with the combination of (only) this package and ComfyUI-Dev-Utils.

**Comment by thefoxfarmer** [comment]
@807502278 I recommend you change the title of this issue to be more descriptive, like "Input/output context menu vanishes when used in conjunction with some other custom nodes"
